docs/run/nodes/nodes.md

```diff
@@ -280,7 +278,7 @@ Ceramic nodes rely on IPFS for networking. IPFS nodes connect to each other usin
 
     Healthchecks can be run against the `HEALTHCHECK_PORT` (port `8011` by default) when `HEALTHCHECK_ENABLED` is `true`.
 
-Additionally, when running IPFS separately from Ceramic, the IPFS API port must be accessible by the Ceramic node. The default API port is `5011`. The IPFS node address will then be passed to Ceramic with a CLI flag `--ipfs-api <ipfs_api_url>`.
+Additionally, when running IPFS the IPFS API port must be accessible by the Ceramic node. The default API port is `5001`. The IPFS node address will then be passed to Ceramic with a CLI flag `--ipfs-api <ipfs_api_url>`, or the `ipfs.host` option in the Ceramic daemon config file.
```

Comment:
with go-ipfs-daemon, the default port is `5011`, not 5001

Also given cli flags are deprecated should we remove that instruction?

Reply:
done.  Also replaced a couple of other references to cli flags with their config file equivalents